server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsRequestBuilder.java

```diff
@@ -40,6 +40,17 @@ public GetSnapshotsRequestBuilder setRepositories(String... repositories) {
         return this;
     }
 
+    /**
+     * Sets slm policy patterns
+     *
+     * @param policies slm policy patterns
+     * @return this builder
+     */
+    public GetSnapshotsRequestBuilder setPolicies(String... policies) {
```

Comment:
Is there a difference between the `setPolicies` and `policies` methods? It seems to me they do the same thing.

Reply:
One is on the builder, one on the normal request instance. This is one mainly just so that we can write nice tests in a fluent manner :)